From 00bfaee64181c100b5f29b89b94c7180b5af363f Mon Sep 17 00:00:00 2001
From: Richard Copley <buster@buster.me.uk>
Date: Sun, 11 Apr 2021 14:40:17 +0100
Subject: [PATCH 1/4] Revert "libstdc++: Make
 coroutine_handle<_Promise>::from_address() noexcept [PR 99021]"

This reverts commit fa183497cf25b604f5b76bc16766f30f5ec7e05b.
---
 libstdc++-v3/include/std/coroutine | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libstdc++-v3/include/std/coroutine b/libstdc++-v3/include/std/coroutine
index 21bdd1b2f04..468d1107557 100644
--- a/libstdc++-v3/include/std/coroutine
+++ b/libstdc++-v3/include/std/coroutine
@@ -197,7 +197,7 @@ _GLIBCXX_BEGIN_NAMESPACE_VERSION
       }
 
     // 17.12.3.2, export/import
-    constexpr static coroutine_handle from_address(void* __a) noexcept
+    constexpr static coroutine_handle from_address(void* __a)
     {
       coroutine_handle __self;
       __self._M_fr_ptr = __a;
-- 
2.31.0.windows.1

